src/main/java/com/acquia/http/HMACFilter.java

```diff
@@ -81,6 +129,24 @@ public void destroy() {
 
     }
 
+    /**
+     * Check if timestamp is within tolerance (900 seconds)
+     * 
+     * @param unixTimestamp
+     * @return non-zero if timestamp is outside tolerance (positive if in the future; negative in the past); otherwise return zero
+     */
+    public int compareTimestampWithinTolerance(long unixTimestamp) {
```

Comment:
I'll change this to protected. This is originally calling HMACUtil, but mockito doesn't allow mocking static method, so I duplicate the code in this class.. HMACServlet is still using HMACUtil ones..
Not sure if this is a good way to do it.. Input?